### PR TITLE
Inject test data on test startup

### PR DESCRIPTION
**Pull request by Sammers21**
A solution for #15

### CODE REVIEW

**Comment by Sammers21** [comment]
@tsegismont, can you review?

**Comment by Sammers21** [comment]
@tsegismont, can please take a look at it?

**Comment by tsegismont** [comment]
@Sammers21 it looks good to me. But before merging we need to get approval from Datastax as their GitHub repo doesn't mention any license (meaning all rights reserved, by default).

**Comment by Sammers21** [comment]
@tsegismont, how we can get their approval? Where can we ask for it?

**Comment by Sammers21** [comment]
ah, I see, you have just asked in twitter

**Comment by tsegismont** [comment]
Yep :)

**Comment by Sammers21** [comment]
@tsegismont, I have just replaced datastax playlist data with auto generated one. Can you review, please?

File: pom.xml
Comment (tsegismont) [inline pom.xml:115]: Where is this used?

Comment (tsegismont) [reply pom.xml:115]: Ha, I see, `randomAlphaString`

File: src/test/java/io/vertx/cassandra/CassandraServiceBase.java
Comment (tsegismont) [inline src/test/java/io/vertx/cassandra/CassandraServiceBase.java]: Since the project depends on `vertx-unit`, you should add a `TestContext` parameter to the method and then use `TestContext#async` and `Async#fail`

The failure will be reported as the test error and then there's no need to log it separately

File: src/test/java/io/vertx/cassandra/CassandraServiceBase.java
Comment (tsegismont) [inline src/test/java/io/vertx/cassandra/CassandraServiceBase.java]: Replace usage of the latch with `TestContext#async(int)`

File: src/test/java/io/vertx/cassandra/CassandraServiceBase.java
Comment (tsegismont) [inline src/test/java/io/vertx/cassandra/CassandraServiceBase.java]: Where is this one used?

**Review by tsegismont — CHANGES_REQUESTED** [review]
A few questions/concerns

File: src/test/java/io/vertx/cassandra/CassandraServiceBase.java
Comment (Sammers21) [reply src/test/java/io/vertx/cassandra/CassandraServiceBase.java]: when calling `CompositeFuture#all` to wait for keyspaces initialization

Comment (tsegismont) [reply src/test/java/io/vertx/cassandra/CassandraServiceBase.java]: Right but is it used in other tests?

File: src/test/java/io/vertx/cassandra/CassandraServiceBase.java
Comment (Sammers21) [reply src/test/java/io/vertx/cassandra/CassandraServiceBase.java]: Done

File: src/test/java/io/vertx/cassandra/CassandraServiceBase.java
Comment (Sammers21) [reply src/test/java/io/vertx/cassandra/CassandraServiceBase.java]: Done

File: src/test/java/io/vertx/cassandra/CassandraServiceBase.java
Comment (Sammers21) [reply src/test/java/io/vertx/cassandra/CassandraServiceBase.java]: For now, it used in `ExecutionTest#tableHaveSomeRows` and actually it is going to be used in https://github.com/Sammers21/vertx-cassandra-client/pull/14

File: src/test/java/io/vertx/cassandra/CassandraServiceBase.java
Comment (tsegismont) [inline src/test/java/io/vertx/cassandra/CassandraServiceBase.java]: Can you fail the context with the failure instead of logging it?

**Review by tsegismont — CHANGES_REQUESTED** [review]

File: src/test/java/io/vertx/cassandra/CassandraServiceBase.java
Comment (Sammers21) [reply src/test/java/io/vertx/cassandra/CassandraServiceBase.java]: Yes, just done

**Comment by Sammers21** [comment]
@tsegismont , can you review one more time, please?

**Review by tsegismont — APPROVED** [review]

**Comment by tsegismont** [comment]
Looks good to me. Thanks Pavel!

**Pull request merged**
